### PR TITLE
Test for i18n

### PR DESCRIPTION
**Pull request by HsiangNianian**

### CODE REVIEW

**Comment by xadhrit** [comment]
no jp or ko in i18n? and I tried adding .en files inside the guide module but seeming like it's not translating in real time. Then I added translated content for all guides . So i've documentation written in english but on site translation not working.

**Comment by xadhrit** [comment]
plus need to change content of about page. My poetry package is broken. I'm reinstalling upgraded version 1.5.1 but seems something off.

**Comment by HsiangNianian** [comment]
> no jp or ko in i18n? and I tried adding .en files inside the guide module but seeming like it's not translating in real time. Then I added translated content for all guides . So i've documentation written in english but on site translation not working.

The thing is, it's not working, so I'm investigating all possible scenarios. That's why I temporarily removed the Japanese (jp) and Korean (ko) translations.

**Comment by HsiangNianian** [comment]
> plus need to change content of about page. My poetry package is broken. I'm reinstalling upgraded version 1.5.1 but seems something off.

Don't worry, I will do my best to fix the document. Rest assured, I look forward to hearing good news from you. If you need any assistance, feel free to ask me :D

File: docs/components/use_locales_map.js
Comment (xadhrit) [inline docs/components/use_locales_map.js:1]: how about using react-i18n?

**Comment by HsiangNianian** [comment]
Thanks so much for the advice. I'll try again after work today, if that doesn't work 🌚 I'll try react-i18n :D

Hsiang Nianian,
此致。
________________________________

**Comment by HsiangNianian** [comment]
webpack: (config) => {
    config.module.rules.push({
      test: /\.(js|jsx|ts|tsx)$/,
      use: {
        loader: 'babel-loader',
        options: {
          presets: ['@babel/preset-env', '@babel/preset-react'],
        },
      },
    });
    return config;
  },

**Comment by HsiangNianian** [comment]
done

**Pull request closed without merging**